docs/api-docs/channels/channel-webhooks.md

```diff
@@ -265,6 +312,32 @@ Web pages payload objects take the form that follows:
  
 For a complete reference of all BigCommerce webhook events and their callback payloads, see [Webhook Events](/api-docs/store-management/webhooks/webhook-events).
 
+## Price list assignments
+
+The following pricelist assignment webhook events fire in response to actions that affect a specific channel on a store:

```

Comment:
make price list one word everywhere unless we have a style thing saying differently.